### PR TITLE
Fixes #1641 : Video,GIF of cloud deploy

### LINKED ISSUE #1641: Missing ASCIICAST / Video guide
**Actual Behaviour**
The install/ deploy steps miss `asciicast`(s) or `video`(s)

**Expected Behaviour**
Steps must be accompanied with either of the above.

**Steps to reproduce it**
Add ASCIICAST/VIDEO of guide

**Would you like to work on the issue?**
Yes

### PR DESCRIPTION
**Pull request by YashKumarVerma**
Fixes #1641 

Changes: 
-Made a complete guide asciinema,  to illustrate deploy steps to google ☁️ 
-Uploaded to asciinema 🔼 
-A youtube video
- A gif
https://asciinema.org/a/156500
https://youtu.be/AKOHR6uLMlY

### CODE REVIEW

**Comment by YashKumarVerma** [comment]
@fossasia/open-event-core

**Comment by geekyd** [comment]
@YashKumarVerma Can we add a GIF instead of video link?

**Comment by YashKumarVerma** [comment]
A gif that long would't be tedious ???
and secondly, the asciicast has features like seek and spped also, making it more easy to follow

**Comment by YashKumarVerma** [comment]
@geekyd

**Comment by geekyd** [comment]
@YashKumarVerma The video is too long. We can add a video but you need to fast forward the video when the build is getting deployed. We do not need to show the logs. Make it under 2 minutes.
Also it would be great if we could add this video to official FOSSASIA youtube channel. @mariobehling What do you think?

**Comment by YashKumarVerma** [comment]
sure @geekyd , i'd update the link. The fast forward controlls are there on the player itself.

**Comment by YashKumarVerma** [comment]
converting to video of about 2-3 minutes

File: docs/INSTALLATION_GOOGLE.md [hunk 1]
```diff
@@ -70,3 +70,9 @@ You can view the application in your web browser using:
 gcloud app browse
 ```
 The application will now open in a new tab in your browser for you to view.
+
+See the entire process step by step
+![GIF](screenshots/open-event-webapp.gif)
+- [Video](https://www.youtube.com/watch?v=AKOHR6uLMlY)
```
Comment (geekyd) [inline docs/INSTALLATION_GOOGLE.md]: Remove the video as GIF & video are same

**Comment by YashKumarVerma** [comment]
@geekyd Done. 😄

**Pull request merged**